lib/full-site-editing/block-templates.php

```diff
@@ -150,18 +179,19 @@ function _inject_theme_attribute_in_content( $template_content ) {
 	$new_content         = '';
 	$template_blocks     = parse_blocks( $template_content );
```

Comment:
Not related to this PR, but I noticed that `parse_blocks` returns a few empty blocks/elements in the array (with `blockName` => `NULL` and other elements empty). Is it just me and is this a known/expected thing?

Reply:
Looks like those are used to catch text between blocks.
See here first block is a template part, then there is an empty line in the code before the group block:
```html
<!-- wp:template-part {"slug":"header","tagName":"header","className":"site-header","layout":{"inherit":true}} /-->

<!-- wp:group {"tagName":"main","layout":{"inherit":true}} -->
......
......

This is represented in "blocks" like this:
```
![image](https://user-images.githubusercontent.com/2256104/113723271-79391180-96f1-11eb-94db-9db0ffadae4a.png)